src/components/common/ProfilePanel.js

```diff
@@ -242,39 +246,41 @@ class ProfilePanel extends React.Component {
     </Button>
   );
 
-  renderButtons = () => {
+  const renderButtons = () => {
     const panelButtons = [];
-    switch (this.props.contentType) {
-    case 'sketches':
-      panelButtons.push(this.renderEditorButton());
-      break;
-    case 'editor':
-    default:
-      panelButtons.push(this.renderSketchesButton());
-      break;
+    switch (
+      props.contentType //props need to be passed contentType
+    ) {
+      case 'sketches':
+        panelButtons.push(renderEditorButton());
+        break;
+      case 'editor':
+      default:
+        panelButtons.push(renderSketchesButton());
+        break;
     }
 
-    panelButtons.push(this.renderSignOutButton());
+    panelButtons.push(renderSignOutButton());
 
     return <div className="panel-buttons">{panelButtons}</div>;
   };
 
-  renderThemeSwitch = () => {
+  const renderThemeSwitch = () => {
     const onToggle = (on) => {
-      this.props.onThemeChange(on ? 'light' : 'dark');
+      props.onThemeChange(on ? 'light' : 'dark');
     };
 
```

Comment:
nit: you can remove `{}` if you want since it's one line arrow function, but not important